src/main/java/com/salesforce/mce/awesolog/S3FixedWindowRollingPolicy.java

```diff
@@ -133,6 +135,14 @@ public void setAwsSecretKey(String awsSecretKey) {
         this.awsSecretKey = awsSecretKey;
     }
 
+    public String getAwsRoleToAssume() {
```

Comment:
are these getters necessary? seems you only need those within this class?

Reply:
Logback uses JavaBeans introspection to analyze properties set on the appender, think custom appender will need both getter and setter methods to find and set these properties

Reply:
Direct except from https://logback.qos.ch/manual/appenders.html: `All properties that follow the setter/getter JavaBeans conventions are handled transparently by logback configurators. `

Reply:
ok, make sense